src/main/java/ggs/srr/service/initdata/InitDataService.java

```diff
@@ -152,14 +153,23 @@ private List<ProjectDto> parseProjects(LinkedHashMap<String, Object> data){
 
     private FtUser parseUser(LinkedHashMap<String, Object> data){
 
+        double level = 0L;
+        Object cursusUsers = data.get("cursus_users");
+        List<LinkedHashMap<String, Object>> usersData = (List<LinkedHashMap<String, Object>>)cursusUsers;
+        System.out.println(usersData);
```

Comment:
```suggestion
        List<LinkedHashMap<String, Object>> usersData = (List<LinkedHashMap<String, Object>>)cursusUsers;
```
디버깅 구문이 들어온 것 같아요! 확인 부탁드립니다.